tests/acceptance/features/apiOcm/share.feature

```diff
@@ -100,6 +100,96 @@ Feature: an user shares resources usin ScienceMesh application
       | folderToShare |
       | textfile.txt  |
 
+
+  Scenario: local user shares resources from project space to federation user
```

Comment:
IMHO,
```suggestion
  Scenario: check activity after sharing the resource from project space of local user to federation user
```
will make the scenario more clear? :thinking:

Reply:
Yeah, makes sense but just following the same pattern as other scenarios in that feature file.

Reply:
sorry but where activity api has been hit?

Reply:
We are not hitting activity api as we're just listing the shares not checking the activity.

Reply:
was just the misleading PR title - which has been changed